src/query/test/parser/ast.go

```diff
@@ -0,0 +1,434 @@
+// Copyright 2015 The Prometheus Authors
```

Comment:
Can we not use the exported types from this package in prometheus itself?  

Just curious on what modifications were made that made this not possible?